static/js/chartHandler.js

```diff
@@ -79,6 +92,8 @@ const format = input => {
     return 'r\u2080'
   }else if(input === 'r_a'){
     return "r\u2090" 
```

Comment:
Correct me if I'm wrong, but wouldn't we need to change these conditionals as well to reflect the "ra" and "r0" values?

Reply:
Great catch! Just changed